support-workers/src/main/scala/com/gu/support/workers/PreviewPaymentSchedule.scala

```diff
@@ -13,7 +13,7 @@ object PreviewPaymentSchedule {
   def apply(
     subscribeItem: SubscribeItem,
     billingPeriod: BillingPeriod,
-    services: Services,
+    zuoraService: ZuoraSubscribeService,
```

Comment:
this only uses "previewSubscribe" but it's not unit tested anyway. if we want to add unit test later we could use a single abstract method trait to make the tests clear.